CHANGELOG.asc

```diff
@@ -6,4 +6,35 @@ Release Notes
 Version 0.1.0 (TBD)
 ~~~~~~~~~~~~~~~~~~~
 
-Changelog to be added once released.
+[source, m2]
+<dependency>
+   <groupId>org.janusgraph</groupId>
+   <artifactId>janusgraph-core</artifactId>
+   <version>0.1.0</version>
+</dependency>
+
+*Tested Compatibility:*
+
+* Cassandra 2.1.9
+* HBase 0.98.8-hadoop2, 1.0.3, 1.1.8, 1.2.4
+* BerkeleyJE 7.3.7
+* Elasticsearch 1.5.1
+* Lucene 4.10.4
+* Solr 5.2.1
+* TinkerPop 3.2.3
+* Java 1.8
+
+*Features added since version Titan 1.0.0:*
+
+* TinkerPop 3.2.3 compatibility
+** Includes update to Spark 1.6.1
+* Query optimizations: JanusGraphStep folds in HasId and HasContainers can be folded in even mid-traversal
+* Support Google Bigtable as a backend over the HBase interface
+* Compatibility with newer versions of backend and index stores
+** HBase 1.2
+** BerkeleyJE 7.3.7
+* Includes a number of bugfixes and optimizations
+
+For more information on features and bugfixes in 0.1.0, see these Github milestones:
+
+* https://github.com/JanusGraph/janusgraph/milestone/1
```

Comment:
If we refer people to this link, can we verify we do have the important issues added in the M1?